Makefile

```diff
@@ -15,6 +15,8 @@ IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
 K8S_NAMESPACE := kube-system
 KUBECONFIG :=
 
+GOFMT_CHECK=$(shell find . -not \( \( -wholename './.*' -o -wholename '*/vendor/*' \) -prune \) -name '*.go' | sort -u | xargs gofmt -s -l)
```

Comment:
You're already doing the gofmt check here. Could that be moved to the actual make target? I.e. have something like `GO_FILES` here and then running `gofmt -s -l $(GO_FILES)` in the make target